src/styles/styles.js

```diff
@@ -698,6 +698,11 @@ const styles = {
         borderRadius: variables.componentBorderRadiusNormal,
     },
 
+    textInputMultiline: {
+        scrollPadding: '23px 0 0 0',
```

Comment:
```suggestion
        scrollPadding: '23px 0 0 0 0',
```

I think we only need top padding. Right?

Reply:
Sorry ignore this, I am on drugs... Lol.